README.md

```diff
@@ -62,7 +63,7 @@ Commits must preserve proper and informative naming, e.g. *'sm6150-common: Add m
     * Must be operational and support same range of satellite types as stock
 * **Storage**
     * Every storage memory must be working, internal and external (like sd cards)
-    * Devices must support encryption. FDE devices should switch to FBE if possible, FBE encryption must be enabled by default
+    * Devices must support encryption. FDE devices should switch to FBE, FBE encryption must be enabled by default
```

Comment:
keep if possible for now as support if getting dropped with A13

Reply:
It is not needed as per 'should' is already a suggestion rather than an enforcement.